core/src/main/java/io/kestra/core/server/ClusterEvent.java

```diff
@@ -0,0 +1,15 @@
+package io.kestra.core.server;
+
+import io.kestra.core.models.HasUID;
+import io.kestra.core.utils.IdUtils;
+
+import java.time.LocalDateTime;
+
+public record ClusterEvent(String uid, EventType eventType, LocalDateTime eventDate, String message) implements HasUID {
```

Comment:
I'm wondering if instead of typing event with an enum (`EventType`); we should not create a sealed `ClusterEvent` interface, with a dedicated class for each event type. The main reason is because I think we  will need to attach some data with events in future. In any case, moving from an enum to a sealed interface is an easy refactoring and can be done later if necessary.

Reply:
It's an interesting idea, in fact I added a message event but I found it weak and added a non-used string in it.

Reply:
But this would need JsonTypeInfo and redoing the switch, I'll let that for a later refactoring ;)